app/navigation/CreateNavigator.scala

```diff
@@ -101,6 +104,8 @@ class CreateNavigatorImpl extends CreateNavigator with CreateAnswerConditions wi
 
 protected trait CreateAnswerConditions {
 
+  protected implicit val appConfig: FrontendAppConfig
```

Comment:
OBSERVATION:  With the change to answers.isCmaAllowed I don't think we need appConfig here anymore.

Reply:
We do - it's a required argument for answers.isCmaAllowed.  Couldn't inject appConfig into UserAnswers as it's a case class.